internal/commands/secrets/inputs.go

```diff
@@ -0,0 +1,107 @@
+package secrets
+
+import (
+	"errors"
+	"fmt"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+// Flag names and usages across the secrets commands
+const (
+	flagName            = "name"
+	flagNameShort       = "n"
+	createFlagNameUsage = "the name of the secret to add to your Realm App"
+	updateFlagNameUsage = "the new name for the secret"
+
+	flagValue            = "value"
+	flagValueShort       = "v"
+	createFlagValueUsage = "the value of the secret to add to your Realm App"
+	updateFlagValueUsage = "the new value for the secret"
+
+	flagSecret            = "secret"
+	flagSecretShort       = "s"
+	updateFlagSecretUsage = "ID or name of the secret to update"
+
+	flagSecrets            = "secret"
+	flagSecretsShort       = "s"
+	deleteFlagSecretsUsage = "set the list of secrets to delete by ID or Name"
+)
+
+type promptFn func([]string) survey.Prompt
```

Comment:
[question] were there any major differences with `secrets` compared to `users` that we couldn't follow a similar `inputs` pattern that was established there?  I don't think we need something like this `promptFn`...

lets work offline to see if im missing something here

Reply:
the main difference is that we can only update _one_ secret but we can delete _many_ secrets 
that changes the `survey.Prompt` that we need to use (MultiSelect, Select) as well as the message 
`promptFn` lets those individual components stay with the corresponding command files (`delete_inputs`, `update_inputs`) while letting `inputs` be a little more generic for any other application (returning one secret, returning many secrets)

Reply:
discussed two possible helpers here, but otherwise I'd push a lot of this logic just to the individual command input `Resolve` functions themselves